src/core/tsi/s2a/s2a_constants.h

```diff
@@ -43,9 +43,9 @@ constexpr size_t kTlsAes128GcmSha256NonceSize = 12;
 constexpr size_t kTlsAes256GcmSha384NonceSize = 12;
 constexpr size_t kTlsChacha20Poly1305Sha256NonceSize = 12;
 
-/** The size (in bytes) of the additional data bytes buffer used for encrypting
- * and decrypting TLS 1.3 records. **/
-constexpr size_t kTlsAdditionalDataBytesSize = 13;
+/** The size of the additional data bytes buffer used for encrypting and
+ *  decrypting TLS 1.3 records. **/
+constexpr size_t kTlsAdditionalDataBytesSize = 5;
```

Comment:
In TLS 1.3, the additional data bytes that are used are the TLS record header, which is 5 bytes long. 

In TLS 1.2, the additional data bytes size is 13, so some record protocols that implement both TLS 1.2 and 1.3 set the additional data bytes size to 13 and then only use 5 of those bytes in the case of TLS 1.3. 

This is what I had originally done, but realized that we could simplify things since we only support TLS 1.3, hence the change from 13 to 5.